shipment_advice/tests/test_shipment_advice_load.py

```diff
@@ -0,0 +1,154 @@
+
+    def test_shipment_advice_load_move_line_not_planned(self):
+        self._in_progress_shipment_advice(self.shipment_advice_out)
+        move = self.move_product_out1
+        wiz_model = self.env["wizard.load.shipment"].with_context(
+            active_model=move.move_line_ids._name,
+            active_ids=move.move_line_ids.ids,
+        )
+        wiz = wiz_model.create({"shipment_advice_id": self.shipment_advice_out.id})
+        self.assertEqual(wiz.move_line_ids, move.move_line_ids)
+        self.assertFalse(wiz.picking_ids)
+        wiz.action_load()
+        # Check planned entries
+        self.assertEqual(wiz.shipment_advice_id, self.shipment_advice_out)
+        self.assertFalse(wiz.shipment_advice_id.planned_picking_ids)
+        self.assertEqual(wiz.shipment_advice_id.planned_pickings_count, 0)
+        self.assertFalse(wiz.shipment_advice_id.planned_move_ids)
+        self.assertEqual(wiz.shipment_advice_id.planned_moves_count, 0)
+        # Check loaded entries
+        self.assertEqual(wiz.shipment_advice_id.loaded_picking_ids, move.picking_id)
+        self.assertEqual(wiz.shipment_advice_id.loaded_pickings_count, 1)
+        self.assertEqual(
+            wiz.shipment_advice_id.loaded_move_line_ids, move.move_line_ids
+        )
+        self.assertEqual(
+            wiz.shipment_advice_id.loaded_move_lines_without_package_count, 1
+        )
+
+    def test_shipment_advice_load_move_line_already_planned(self):
+        move = self.move_product_out1
+        self._plan_records_in_shipment(self.shipment_advice_out, move)
+        self._in_progress_shipment_advice(self.shipment_advice_out)
+        wiz = self._load_records_in_shipment(
+            self.shipment_advice_out, move.move_line_ids
+        )
+        self.assertEqual(wiz.move_line_ids, move.move_line_ids)
+        self.assertFalse(wiz.picking_ids)
+        # Check planned entries
+        self.assertEqual(wiz.shipment_advice_id, self.shipment_advice_out)
+        self.assertEqual(wiz.shipment_advice_id.planned_picking_ids, move.picking_id)
+        self.assertEqual(wiz.shipment_advice_id.planned_pickings_count, 1)
+        self.assertEqual(wiz.shipment_advice_id.planned_move_ids, move)
+        self.assertEqual(wiz.shipment_advice_id.planned_moves_count, 1)
+        # Check loaded entries
+        self.assertEqual(wiz.shipment_advice_id.loaded_picking_ids, move.picking_id)
+        self.assertEqual(wiz.shipment_advice_id.loaded_pickings_count, 1)
+        self.assertEqual(
+            wiz.shipment_advice_id.loaded_move_line_ids, move.move_line_ids
+        )
+        self.assertEqual(
+            wiz.shipment_advice_id.loaded_move_lines_without_package_count, 1
+        )
+
+    def test_shipment_advice_load_moves_different_pack(self):
+        move = self.move_product_out1
+        move_package_ids = (
+            self.move_product_out2.move_line_ids.ids
+            + self.move_product_out3.move_line_ids.ids
+        )
```

Comment:
I would align the variable name with what it represents actually:
```suggestion
        move_package_ids = (
            self.move_product_out2
            + self.move_product_out3
        )
```